### PR TITLE
Tentative support for OBDR on ppc64le (issue #1868)

### PR DESCRIPTION
**Pull request by jsmeix**
* Type: **Bug Fix** **Enhancement**

* Impact: **Normal**

* Reference to related issue (URL):
https://github.com/rear/rear/issues/1868

* How was this pull request tested?
I neither have any tape device nor can I use OBDR
so that I cannot test or reproduce anything that is specific for OBDR.

* Brief description of the changes in this pull request:
Initial tentative support for OBDR on ppc64le
via a new `output/OBDR/Linux-ppc64le` sub-directory
that contains currently only two symlinks to existing scripts
<pre>
output/OBDR/Linux-ppc64le/300_create_grub2.sh -> ../../ISO/Linux-ppc64le/300_create_grub2.sh
output/OBDR/Linux-ppc64le/800_create_isofs.sh -> ../../ISO/Linux-ppc64le/800_create_isofs.sh
</pre>
cf. https://github.com/rear/rear/issues/1868#issuecomment-405613322

### CODE REVIEW

**Comment by jsmeix** [comment]
@schabrolles
I have an idea how you could do some kind of
"blind smoke tests" on real PPC64 and PPC64LE systems
that could indicate if this changes here could work at all:

Specify `OUTPUT=OBDR` and then run `rear -s mkrescue`
on a PPC64 system and on a PPC64LE system and
compare what scripts would get run on each of them.

When the scripts that would get run on PPC64LE
look reasonable to you compared to the the scripts
that would get run on PPC64 we can assume
this changes here could work at all.

In this case I would "just blindly" merge this pull request
because the two more symbolic links for OBDR on PPC64LE
cannot cause a regression elsewhere and if OBDR still
does not work in parctice on PPC64LE it is also no regression
compared to what it was before this pull request.

Simply put:
I would like to merge this pull request soon so that
our SUSE partner who tests OBDR on PPC64LE on SLES12
can try out our ReaR GitHub master code with real OBDR hardware.

**Comment by schabrolles** [comment]
@jsmeix here is the diff of `rear -s mkrescue` on ppc64 and ppc64le. Looks good.
```
diff rear-output.ppc64 rear-output.ppc64le               
1c1
< Source conf/Linux-ppc64.conf
---
> Source conf/Linux-ppc64le.conf
134c134
< Source pack/Linux-ppc64/300_copy_kernel.sh
---
> Source pack/Linux-ppc64le/300_copy_kernel.sh
143c143
< Source output/OBDR/Linux-ppc64/300_create_yaboot.sh
---
> Source output/OBDR/Linux-ppc64le/300_create_grub2.sh
146c146
< Source output/OBDR/Linux-ppc64/800_create_isofs.sh
---
> Source output/OBDR/Linux-ppc64le/800_create_isofs.sh
154d153
< 
```
